### PR TITLE
Allergy clinical status update

### PR DESCRIPTION
**Pull request by au2**
#### What does this PR do?
 
Removes Effective Time based update of allergy clinical status and jolt implements and puts tests for Allergy Status template based update of allergy clinical status.

#### Related JIRA tickets:

UPMCFHIR-341

#### How should this be manually tested?

Take a look at the sample files to see if the values are adequate.

#### Background/Context:

NA

#### New JIRA tickets for clinical review required?

NA

### CODE REVIEW

File: src/test/java/tr/com/srdc/cda2fhir/testutil/JoltUtil.java [hunk 1]
```diff
@@ -135,29 +135,6 @@ public void copyReferences(Map<String, Object> joltResult) {
 		}
 	}
 
-	private static class OrganizationInfo extends ResourceInfo {
```
Comment (mmccall) [inline src/test/java/tr/com/srdc/cda2fhir/testutil/JoltUtil.java:138]: looks like you changed some stuff with organization and patient as well, was this intentional? Were you fixing some of that work left to do?

**Review by mmccall — COMMENTED** [review]
I think you changed a few other things in here besides this, so long as it was intentional (looks like Jolt stuff). 

I ran the files through, and the Epic files make sense to me. However, the Cerner files are now not getting clinical status or verification status. I'm actually okay with us not inferring Clinical Status for them, but verification is required per the spec (It looks like this requirement was relaxed in R4). The cerner files have a `<statusCode nullFlavor="UNK"/>` on allergy problem act where we are looking.  I'd propose when we encounter this, we set verificationStatus to `unconfirmed`.  If you agree, you can either fold it in here, or ticket it out and get this merged in, just let me know what you think.

**Comment by au2** [comment]
Yeah, that makes sense. I will write a ticket. Null flavor/defaulting gets interesting with Jolt.

Other jolt stuff is intentional or was intentional for the other branch but accidentally done here. It is ok to leave them; I need them for Jolt fixes/cleaning.

**Comment by au2** [comment]
I created UPMCFHIR-344. Just to clarify we are looking at AllergyStatus template currently. We can default to unconfirmed for that one but looks like you were thinking we were using status code on AllergyAct level which is like two level up. LMK if that changes things. Otherwise I can move this whenever you approve.

**Comment by mmccall** [comment]
> I created UPMCFHIR-344. Just to clarify we are looking at AllergyStatus template currently. We can default to unconfirmed for that one but looks like you were thinking we were using status code on AllergyAct level which is like two level up. LMK if that changes things. Otherwise I can move this whenever you approve.

We are looking at the allergyStatus template for clinical status, and AllergyAct status for verificationStatus (to my understanding). In Cerner neither are set, but the allergyAct status is unknown, so we could set the verificationStatus to unconfirmed. If what I just said sounds right to you, go ahead and merge, if not shoot me a message :).

**Review by mmccall — APPROVED** [review]
please double check my response to your comment, and if we are on the same page go ahead and merge.

**Pull request merged**